README.md

```diff
@@ -15,16 +11,24 @@ This playbook will install:
 - Drupal
 - Apigee Kickstart Drupal profile
 
-The topology can be one host with everything, or two hosts where one is the 
-database, and the other is the web server which contains the Drupal site.
+This playbook is intended to be a starting point and not a "one size fits all" playbook. We recommend you fork this project and make changes to the `playbook.xml` and other files to fit your specific needs.
+
+The topology can be one host with everything, or two hosts where one is the database, and the other is the web server which contains the Drupal site.
+
+## Supported Platforms
```

Comment:
## Target Platforms
so as to not imply an expectation of support?

Reply:
changed